packages/runtime/container-runtime/src/inboundBatchAggregator.ts

```diff
@@ -82,55 +40,29 @@ class ScheduleManagerCore {
 		private readonly getClientId: () => string | undefined,
 		private readonly logger: ITelemetryLoggerExt,
 	) {
-		// Listen for delta manager sends and add batch metadata to messages
-		this.deltaManager.on("prepareSend", (messages: IDocumentMessage[]) => {
```

Comment:
Note that this logic is not needed any more because container runtime handles this before it sends the batch to delta manager. See PR description for more details.